pymc_marketing/mmm/delayed_saturated_mmm.py

```diff
@@ -1391,11 +1365,10 @@ def add_lift_test_measurements(
             target_transform=self.target_transformer.transform,
         )
         with self.model:
-            add_logistic_empirical_lift_measurements_to_likelihood(
+            add_lift_measurements_to_likelihood(
                 df_lift_test=df_lift_test_scaled,
-                # Based on the model
-                lam_name="lam",
-                beta_name="beta_channel",
+                variable_mapping=self.saturation.variable_mapping,
+                saturation_function=self.saturation.function,
```

Comment:
@juanitorduz This is how lift tests are supported now so no need for hard-coded versions for each function. That's why they were deleted from lift_test.py

Reply:
Fantastic!